heron/common/src/java/com/twitter/heron/common/basics/TypeUtils.java

```diff
@@ -87,6 +87,17 @@ public static ByteAmount getByteAmount(Object o) {
     }
   }
 
+  public static ByteAmount getByteAmountMB(Object o) {
```

Comment:
@mycFelix Bill is on vacation so he may not have spare time to reply you. We can definitely add `ByteAmountUnit` as a enum class to keep it simple. Another approach is to make [`2048`](https://github.com/twitter/heron/pull/1728/files#diff-7a5b4ac4a4ebcf917aaa8c34416e02f5R27) into [`2048 * 1024`]. We sacrifice some clarity here but can avoid extending current APIs. Thoughts? @ashvina